compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1466,6 +1466,7 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
             }
             if def_kind.has_codegen_attrs() {
                 record!(self.tables.codegen_fn_attrs[def_id] <- self.tcx.codegen_fn_attrs(def_id));
+                record!(self.tables.codegen_instance_attrs[def_id] <- self.tcx.codegen_fn_attrs(def_id));
```

Comment:
This does work (for now), but is highly suspicious. I feel like this really should be using `self.tcx.codegen_instance_attrs`, but we don't have the `InstanceKind` available here.